conf/vmware.yaml.template

```diff
@@ -1,33 +1,31 @@
 VMWARE:
   # Vmware to be added as a compute resource
-  # vmware vcenter URL, e.g. example.com
+  # vcenter: vmware vcenter URL
```

Comment:
If you mean this as example, ok, but it's actually uppercase, right? Why not have the example uppercase as well?

Reply:
Yeah, updated

Reply:
```
trigger: test-robottelo
pytest: tests/foreman/cli/test_computeresource_vmware.py -k test_positive_provision_vmware
provisioning: true
```